routes/api/bookin.js

```diff
@@ -95,7 +95,7 @@ router.get("/history", passport.authenticate("jwt", { session: false }), (req, r
                             });
 
                             // Check if we've finished packing ALL the history objects
-                            if (historyPack.length === bookings.length) {
+                            if (historyPack.length === bookings.length) return res.status(200).send(historyPack);
```

Comment:
what is this?

Reply:
This already existed before, and when I sent the previous PR, somehow the code after if statement "return res.status(200).send(historyPack);" was deleted. So I put it back

Reply:
which one should I keep?
if (historyPack.length === bookings.length) {
or
 if (historyPack.length === bookings.length) return res.status(200).send(historyPack);

Reply:
your PR has nothing to do with that file... how did you even find that something changed there?

I might have been part of a different PR and after you merged dev-general into your branch, it took over that change

Reply:
Remove this from the line:
'return res.status(200).send(historyPack);'